i18n/config.json

```diff
@@ -1,439 +1,439 @@
 [
   {
     "code": "en",
-    "hrefLang": "en",
```

Comment:
Why are we deprecating hrefLang 🤔?

Looks like we aren't using them properly, though we should:

<img width="1425" alt="Screenshot 2023-05-30 at 11 07 13" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/47bd6e01-156f-459d-975d-07b7ecce9452">

cc: @wackerow @pettinarip

Reply:
Yea, we are not using it right now. I don't think its a problem to lose this.

On the other side, the plugin that we are using is [taking care of `hrefLang` in theory](https://github.com/microapps/gatsby-plugin-react-i18next/blob/master/src/Link.tsx#LL25C14-L25C14).